master/internal/stream/system.go

```diff
@@ -98,9 +101,10 @@ func newDBListener(address, channel string) (*pq.Listener, error) {
 // NewPublisherSet constructor for PublisherSet.
 func NewPublisherSet() *PublisherSet {
 	return &PublisherSet{
-		DBAddress: "postgresql://postgres:postgres@localhost/determined?sslmode=disable",
-		Trials:    stream.NewPublisher[*TrialMsg](),
-		Metrics:   stream.NewPublisher[*MetricMsg](),
+		DBAddress:   "postgresql://postgres:postgres@localhost/determined?sslmode=disable",
```

Comment:
Thank you for wiring this through here! It made it so much easier to get the DB URL from main populated throughout the whole streaming updates system!